cmake/public/cuda.cmake

```diff
@@ -267,6 +267,7 @@ if(CAFFE2_USE_CUDNN)
     set_property(
         TARGET caffe2::cudnn PROPERTY INTERFACE_LINK_LIBRARIES
         "${CUDA_TOOLKIT_ROOT_DIR}/lib64/libculibos.a" dl)
+    set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--exclude-libs,libcudnn_static.a")
```

Comment:
This would add this option to all project that include this file.
Please use `set_property(TARGET` or `target_set_property` to limit this change to only targets that include `caffe2:cudnn`